.vscode/settings.json

```diff
@@ -1,5 +1,5 @@
 {
-    "python.pythonPath": "/home/adam/.local/share/virtualenvs/Malcolm-next-EDaiGMpS/bin/python3",
```

Comment:
Mayyyybe don't change my config files. This may need to be in gitignore...

Reply:
It just changes automatically when I run the code. I think putting it the .gitignore might be for the best.

Reply:
Then you need to do that, and revert the change.